### PR TITLE
Add playing card, hand partials

### PR DESCRIPTION
**Pull request by keiranking**
This PR adds partials for playing cards and hands of playing cards.

Problem:
Card game rules are hard to understand without examples. But inserting an example card hand in a Markdown file is not easy.

Trello:
https://trello.com/c/GI2DaOPE
https://trello.com/c/eeBNOYZZ

## After

screen shot 2019-03-01 at 3 21 46 pm

### CODE REVIEW

**Comment by keiranking** [comment]
[New playing card component, partial](https://trello.com/c/GI2DaOPE/3-new-playing-card-component-partial)

**Comment by keiranking** [comment]
[New card-hand component](https://trello.com/c/eeBNOYZZ/4-new-card-hand-component)

**Comment by keiranking** [comment]
[New mini-card-hand component for showing examples in game rules](https://trello.com/c/8h4ogvpQ/11-new-mini-card-hand-component-for-showing-examples-in-game-rules)

File: source/games/kalooki.html.md.erb [hunk 1]
```diff
@@ -10,9 +10,28 @@ The objective of the game is to have the lowest score.
...
+} %>
+
+<%= partial "partials/hand", locals: {
+  caption: "A run is 4 or more suited cards with consecutive ranks.",
+  cards: [
+    "8 of spades",
+    "9 of spades",
+    "10 of spades",
+    "J of spades"
+  ]
```
Comment (danbee) [inline source/games/kalooki.html.md.erb]: This is very cool!

File: source/games/kalooki.html.md.erb [hunk 2]
```diff
@@ -10,9 +10,28 @@ The objective of the game is to have the lowest score.
 
 Each game has 9 rounds. Each round requires players to collect a unique combination of sets and runs.
 
-A set is a collection of 3 or more cards of the same rank, disregarding suit.
+<%= partial "partials/hand", locals: {
+  caption: "A set is 3 or more cards of the same rank, disregarding suit.",
+  cards: [
+    "4 of diamonds",
+    "4 of hearts",
+    "4 of clubs"
```
Comment (tysongach) [inline source/games/kalooki.html.md.erb]: Tossing a trailing comma on this last line in the array is a nice touch to keep future diffs clean, if you ever add an item.

**Pull request merged**